### PR TITLE
Make SQ status context configurable

### PR DESCRIPTION
**Pull request by kargakis**
/cc @spxtr

### CODE REVIEW

**Comment by spxtr** [comment]
/assign @cjwagner 

Do you also need to touch `mungegithub/submit-queue/deployment.yaml` to add the argument? @cjwagner should know for sure.

File: mungegithub/submit-queue/deployment/contrib/configmap.yaml [hunk 1]
```diff
@@ -32,6 +32,9 @@ data:
   submit-queue.chart-url: ""
   submit-queue.history-url: ""
   submit-queue.batch-url: ""
+  # It doesn't seem that this is correct even though the contrib submit queue
+  # today points to this url.
+  submit-queue.context-url: https://submit-queue.k8s.io
```
Comment (cjwagner) [inline mungegithub/submit-queue/deployment/contrib/configmap.yaml]: This should be https://contrib.submit-queue.k8s.io  It looks like the links the contrib submit queue currently uses for its status context don't work.

**Comment by kargakis** [comment]
> Do you also need to touch mungegithub/submit-queue/deployment.yaml to add the argument? @cjwagner should know for sure.

Yeah, forgot to update that since I am using a different manifest, sorry :) Fixed!

File: mungegithub/submit-queue/deployment/contrib/configmap.yaml (hunk 1 above)
Comment (kargakis) [inline mungegithub/submit-queue/deployment/contrib/configmap.yaml]: done

**Pull request merged**